pybossa/data_access.py

```diff
@@ -158,6 +159,19 @@ def ensure_data_access_assignment_from_form(obj, form):
     obj['data_access'] = access_levels
 
 
+@when_data_access()
+def ensure_annotation_config_from_form(obj, form):
+    obj['annotation_config'] = obj.get('annotation_config', {})
+    obj['annotation_config']['amp_store'] = form.amp_store.data
+    obj['annotation_config']['amp_pvf'] = '' if not form.amp_store.data else form.amp_pvf.data or ''
```

Comment:
As pvf w/o opt-in has no meaning, throwing away pvf was to avoid one extra step of throwing validation error and user to explicitly cleanup when they opt out. Let me know if i miss something and I can make changes accordingly.